velox/functions/sparksql/String.h

```diff
@@ -1130,4 +1130,49 @@ struct ReplaceFunction {
   }
 };
 
+template <typename T>
+struct FindInSetFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<int32_t>& result,
+      const arg_type<Varchar>& str,
+      const arg_type<Varchar>& strArray) {
+    int32_t index = 1;
+    int32_t lastComma = -1;
+
+    if (str.find(',') != std::string::npos) {
```

Comment:
should be `std::string_view(str).find(..`?

Reply:
yes, it will be fixed a moment